index.src.html

```diff
@@ -616,6 +623,15 @@ <h5 id="algorithm-create-cred" algorithm>`[[Create]]` internal method</h5>
         a user has just clicked "sign-in" for example, then they won't be surprised or confused to
         see a [=credential chooser=] if necessary.
 
+    :   <dfn>conditional</dfn>
+    ::  Discovered credentials are presented to the user in a non-modal dialog along with an
```

Comment:
This doesn't say anything about what happens when there are no discoverable credentials.

Perhaps something like.

```
If no credentials are discovered, the user agent may prompt the user to take action in a way that depends on
the type of credential, for example, to insert a device containing credentials. The `get()` method MUST NOT
return immediately with `null` to avoid revealing the lack of applicable credentials to the relying party.
```